src/xrEngine/xr_input_xinput.cpp

```diff
@@ -522,6 +522,13 @@ keyboard_key* KeynameToPtr(pcstr name)
 void GetActionAllBinding(pcstr action, char* dst_buff, int dst_buff_sz)
 {
     int action_id = ActionNameToId(action);
+    if (action_id == kNOTBINDED)
```

Comment:
Maybe, better make nifty-counters on StringTable()? It also happens sometimes on Linux
(see GetToken() in console_commands.cpp)

Reply:
@eagleivg, yes, you did misunderstand the purpose of the patch =)